program/dashboard/Dashboard.md

```diff
@@ -18,17 +18,21 @@ PC = Project Closed
-| FORCE-RISCV  		|  PL | Verification                  | Jingliang Wang 					| TBD     		| force-riscv     | CORE-V Cores 	| NONE       	| 20.09.28 (A)	| 20.10.26 (A) 		| 20.11.23 (P)		| https://github.com/openhwgroup/core-v-docs/blob/master/program/FORCE-RISCV%20ISG%20project%20proposal.md								|
-| CVA6  		|  PPL | Verification, Cores           | Jérôme Quévremont 				| TBD     		| cva6       	  | CORE-V Cores 	| 22416       	| 20.09.28 (A)	| TBD 		| TBD		| https://github.com/openhwgroup/core-v-docs/blob/master/program/CVA6%20preliminary%20project%20proposal.md |
-| CORE-V GNU GCC TOOLS  |  PPL | SW                            | Jessica Mills 				| Jessica Mills     		| corev-gcc       | not under EF 	| n/a       	| 20.10.05 (P)	| TBD	 	| TBD 		|  https://github.com/jeremybennett/core-v-docs/blob/jpb-gnu-tools-ppl/program/core-v-gnu-tools-ppl.md 	|							 	|
-| CORE-V MCU SOC (ARNOLD II)  	|  PPL | HW, Verification              | timsaxe, davideschiavone, 	hpollittsmith   |  TBD     		| TBD       	  | CORE-V Cores 	| NONE       	| 20.10.05 (P)	| TBD 		| TBD		| https://github.com/openhwgroup/core-v-docs/blob/master/program/Preliminary%20project%20proposal%20for%20MCU%20SoC.md						 	|	
+| CV32E40P 	  | PPA     	| Verification,  Cores 		| [Mike Thompson](https://github.com/orgs/openhwgroup/people/MikeOpenHWGroup),  [Davide Schiavone](https://github.com/orgs/openhwgroup/people/davideschiavone), [Arjan Binks](https://github.com/orgs/openhwgroup/people/Silabs-ArjanB)  	| [Mike Thompson](https://github.com/orgs/openhwgroup/people/MikeOpenHWGroup) (verif) | [cv32e40p](https://github.com/openhwgroup/cv32e40p)        | CORE-V Cores      	| [22444](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22444), [22415](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22415) 	| grandfathered | grandfathered	| grandfathered	| [core-v cv32e40p readme](https://github.com/openhwgroup/cv32e40p/blob/master/README.md)		|
+| CORE-V MCU FPGA | PPA         | HW    	                | [Hugh Pollitt Smith](https://github.com/orgs/openhwgroup/people/hpollittsmith)                             | 			| [core-v-mcu](https://github.com/openhwgroup/core-v-mcu)      | CORE-V Cores      	| NONE 		| grandfathered | grandfathered	| grandfathered	| [core-v mcu readme](https://github.com/openhwgroup/core-v-mcu/blob/master/README.md)  	|                
+| CORE-V-IDE-CDT  | PPL       	| SW                            | [Alexander Fedorov](https://github.com/orgs/openhwgroup/people/ruspl-afed) 				| TBD      	| [core-v-ide-cdt](https://github.com/openhwgroup/core-v-ide-cdt)  | CORE-V Cores 	| NONE       	| 20.08.31(A) 	| TBD 	| TBD		| [core-v-ide-cdt ppl](https://github.com/openhwgroup/core-v-docs/blob/master/program/CORE-V%20IDE%20prelminary%20project%20proposal.md)			|

```

Comment:
I had in my mind that Alexander was also Project Manager for this. If not, then there is an action on the SW TG to find a Project Manager